src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -996,13 +1003,9 @@ static int xocl_init_drm_mm(struct xocl_drm *drm_p, struct xocl_mm *xocl_mm,
 		goto error;
 	}
 
+	/* Initialize with max and min possible value */
 	drm_mm_init(xocl_mm->mm, mm_start_addr, (mm_end_addr - mm_start_addr));
-	xocl_mm->start_addr = mm_start_addr;
-	xocl_mm->end_addr = mm_end_addr;
-	xocl_mm->m_count = m_count;
-
-	xocl_info(drm_p->ddev->dev, "drm_mm_init called for the available memory range"
-			" <%llx - %llx>", mm_start_addr, mm_end_addr);
+	xocl_info(drm_p->ddev->dev, "drm_mm_init called for the maximum memory range possible");
```

Comment:
Why deleted <%llx - %llx>?

Reply:
We are initializing the max range of memory. Hence, not specifically define any range